source/retryable-reads/tests/changeStreams-client.watch.yml

```diff
@@ -4,14 +4,11 @@ collection_name: &collection_name "coll"
 data:
     - {_id: 1, x: 11}
 
-minServerVersion: '4.1.7' # 4.2
+minServerVersion: '4.0'
 
```

Comment:
Missing topology (I think only in this file).

Reply:
Sorry, fixed! (Some weird shenanigans with my editor)